io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,183 @@
+          "sha256": "85cf747b5923f18a8d0f37d0cef647636d70ddfeda5de49ea019e7e440f9e4e0"
+        },
+        {
+          "type": "file",
+          "dest-filename": "lpub3dldrawunf.zip",
+          "url": "https://github.com/trevorsandy/lpub3d_libs/releases/download/v1.0.1/lpub3dldrawunf-01-04-21.zip",
+          "sha256": "f0266006c4b7114e4f4719801696fbe8780cf9d767e948fb6478c752989cbee8"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/trevorsandy/lpub3d_libs/releases/download/v1.0.1/tenteparts.zip",
+          "sha256": "7e1a68490b1be11bddcfb3180151b42ffbb85ce3aa8ec3f7431bee5c04cc1ac5"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/trevorsandy/lpub3d_libs/releases/download/v1.0.1/vexiqparts.zip",
+          "sha256": "e2f4114c8ff6bfe7e5a43b6c88fb47f32328ddb147578456ec0825aa76803b20"
+        }
+      ],
+      "post-install": [
+        "mv -f complete-20-03.zip ${LP3D_3RD_PARTY_DIR}/complete.zip",      
+        "mv -f lpub3dldrawunf-01-04-21.zip ${LP3D_3RD_PARTY_DIR}/lpub3dldrawunf.zip",
+        "mv -f tenteparts.zip ${LP3D_3RD_PARTY_DIR}/",
+        "mv -f vexiqparts.zip ${LP3D_3RD_PARTY_DIR}/"
+      ]
+    },
+    {
+      "name": "ldglite",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "CONFIG+=3RD_PARTY_INSTALL=/stage/lpub3d_linux_3rdparty/ldglite_renderer",
+        "CONFIG+=release",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldglite.git",
+          "tag": "v1.3.6",
+          "commit": "140865cb85b17eac1abe613066c9364db1e57650"
+        }
+      ],
+      "post-install": [
+        "mv -f ${LP3D_3RD_PARTY_DIR}/ldglite_renderer/* ${LP3D_3RD_PARTY_DIR}/"
+      ]
+    },
+    {
+      "name": "ldview",
+      "buildsystem": "qmake",
+      "config-opts": [
+        "CONFIG+=3RD_PARTY_INSTALL=/stage/lpub3d_linux_3rdparty/ldview_renderer",
+        "CONFIG+=release",
+        "CONFIG+=BUILD_CUI_ONLY",
+        "CONFIG+=USE_SYSTEM_LIBS",
+        "CONFIG+=BUILD_TINYXML",
+        "CONFIG+=BUILD_GL2PS",
+        "CONFIG+=BUILD_CHECK"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/trevorsandy/ldview.git",
+          "branch": "qmake-build",
+          "commit": "16b44f3658690294b1faf13778e3654d443e3751"
+        }
+      ],
+      "post-install": [
+        "mv -f ${LP3D_3RD_PARTY_DIR}/ldview_renderer/* ${LP3D_3RD_PARTY_DIR}/"
+      ]
+    },
+    {
+      "name": "povray",
+      "buildsystem": "simple",
+      "build-commands": [
+        "cd unix && ./prebuild3rdparty.sh",
+        "./configure COMPILED_BY=\"Trevor SANDY <trevor.sandy@gmail.com> using FlatHub for LPub3D.\" --prefix=/stage/lpub3d_linux_3rdparty/povray_renderer LPUB3D_3RD_PARTY=yes --with-libsdl2 --with-boost-libdir=${FLATPAK_DEST}/lib --enable-watch-cursor",
```

Comment:
I think you should directly build in in the prefix (`${FLATPAK_DEST}`).

Reply:
> usually branch + commit is a bad idea because branch are moving targets.

I don't understand you point, are you saying a commit is less of a moving target ? 

I'll use commit if there's going to be problems referencing both.

Cheers,

Reply:
> usually branch + commit is a bad idea because branch are moving targets.

Additionally, I intuitively thought it would be necessary to indicate the branch, if not master, to properly resolve the commit. 

Cheers,